### ISSUE TITLE
Resubmit Sample Missing from Files submitted from Archives (.MSG,.ZIP,etc)

### ISSUE THREAD

**Issue by nicpenning** [cuckoosandbox/cuckoo#1865]
I tried to resubmit a file that was from an archive but the Download and Resubmit Sample options were missing.

Is this something that could be added?

Let me know if you have any other questions!

Thanks!

![image](https://user-images.githubusercontent.com/5582679/30878164-a64e08ce-a2c1-11e7-9718-04207c7fb9c8.png)

**Comment by jbremer** [comment]
Yes, somewhere on the todo list, but implementation wise takes a bit effort. Thanks for feedback.

**Comment by nicpenning** [comment]
Great, thanks for the response.
